src/Participant.js

```diff
@@ -0,0 +1,183 @@
+import {
+  getLocalIsSubscribedToTrack,
+  getRemoteTrack,
+} from './shared-with-pluot-core/selectors';
+
+// Adds tracks to daily-js Participant object.
+export function addTracks(p, prevP) {
```

Comment:
This is mostly a rename of `matchParticipantTracks`, with the contents mostly staying the same (except for some minor code simplification by using the new `getRemoteTrack` selector method). See https://github.com/daily-co/pluot-core/pull/2241 for an explanation of why I didn't yet muck with the tracks themselves in the participants objects.